uiowa/uiowa
# Tippie News Migration

Resolves https://github.com/uiowa/uiowa/issues/4725

# How to test
- Sync tippie.uiowa.edu locally in dev desktop from uiowa703 (db name is tippie_uiowa_edu)
- git checkout tippie_migrate && git pull
- blt ds --site=tippie.uiowa.edu
- drush @tippie.local en tippie_migrate -y && drush @tippie.local uli /admin/config/sitenow/sitenow-migrate
- Complete the form following the help text
- drush @tippie.local ms
- drush @tippie.local mim --all
- Check that images are pulled over.
- Check that you get a content blocks report while running the migration. Last time I checked it was ~77.
- Check that departments, featured research and type fields are being migrated as tags with [some replacements](https://github.com/uiowa/uiowa/files/8654138/News.Tag.Audit_Tippie.xlsx)
- Check that news writer maps to organization and if there is a source title, it is appended to organization.
- Check that if source link is provided, it is only the link and no title.
